### DISCUSSION TITLE
Nextcloud Talk

### DISCUSSION THREAD

**Discussion by w202mg** [AdguardTeam/AdGuardHome#3922] [General]
Hello together,
Yesterday my internet provider was down couple of times. Today I'm facing some strange issue. Once the protection is enabled or my self hosted Nextcloud Talk (on the phone) or both, Nextcloud and Nextcloud Talk, are down. After switching the protection off it runs again, after switched on mostly I can open Nextcloud, but after I try to open Talk both are going down. I tried to roll back the system to the last running condition and to disable the filters but no result. Please help, I'm running out of ideas.

Using AdGard Home with unbound.

Thank you!

**Comment by w202mg** [comment]
Попробую так

Привет всем,
Вчера мой интернет-провайдер пару раз падал. Сегодня я столкнулся с какой-то странной проблемой. Как только я включаю защиту, либо Nextcloud Talk (на телефоне), либо оба, Nextcloud и Nextcloud Talk, не работают (на домашнем сервере). После отключения защиты они снова работают. После того, как я пытаюсь открыть Talk, оба падают. Я пытался откатить систему к последнему работающему состоянию и отключить фильтры, но безрезультатно. Пожалуйста, помогите, у меня уже нет идей.

Использую AdGard Home с unbound.

Спасибо!

**Reply by ainar-g** [reply]
Добрый вечер!  А на странице журнала есть при этом упоминания заблокированных запросов?  Возможно, один из фильтров блокирует работу, и надо добавить правила-исключения.

И ещё, укажите, пожалуйста, версию.

**Reply by w202mg** [reply]
> 

Добрый день! На странице журнала не появляются заблокированные запросы. Версия 0.106. Я пробовал отключать фильты полностью и также отдельно отключить защиту. После отключения защиты Nexcloud и Talk начинают работать, потом опять падают. Если меняю ДНС сервер в рутере на любой другой (не через AdGuard), все работет. Проблема появилась после того, как у провайдера пару раз подряд упал интернет.

**Reply by ainar-g** [reply]
Вы пробовали бета-версии релиза 0.107?  Сам релиз должен выйти в ближайшие дни, и там много улучшений по части стабильности работы, особенно с использованием DNS-over-HTTPS апстримов.

**Reply by w202mg** [reply]
> 
Пока нет, хотел дождаться релиза. Ладно, подожду, там отпишусь. Спасибо!
